### PR TITLE
fix:  inapropriate translation with "前端框架簡介"

### LINKED ISSUE #7994: Issue with "前端框架簡介": Inappropriate translation
## What page(s) did you find the problem on?

https://developer.mozilla.org/zh-TW/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Introduction

## Specific page section or heading?

一段簡短的歷史 (A Brief History)

## What is the problem?

the original content 

> When JavaScript debuted in 1996, it added occasional interactivity and excitement to a web that was, up until then, composed of static documents. The web became not just a place to read things, but to do things. 

had been translated to

> 在 JavaScript 誕生的 1996，它的作用就只有針對網頁，提供些許的互動和興奮。但之後網路漸漸從拿來看，變成拿來用了。

`  在 JavaScript 誕生的 1996，它的作用就只有針對網頁，提供些許的互動和興奮。 ` had not translated the full meaning of original content and might be grammatically incorrect in Chinese.

## What did you expect to see?

> When JavaScript debuted in 1996, it added occasional interactivity and excitement to a web that was, up until then, composed of static documents. The web became not just a place to read things, but to do things. 

should be translated to: 

> 在 JavaScript 誕生的 1996 年，它只有為當時由靜態文檔組成的網頁提供交互性和興奮感的作用。但之後網路漸漸從拿來看，變成拿來用了。

## Did you test this? If so, how?

N/A

### PR DESCRIPTION
**Pull request by devRickyyy**
Fixes: #7994

### CODE REVIEW

File: files/zh-tw/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ slug: Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Introduction
 
 ## 一段簡短的歷史
 
-在 JavaScript 誕生的 1996，它的作用就只有針對網頁，提供些許的互動和興奮。但之後網路漸漸從*拿來看*，變成*拿來用*了。JavaScript 慢慢地紅了起來，JavaScript 開發者也開始針對自己碰上的問題，寫出了能解決問題的工具、接著包成能複用的工具包。他們就能把這個被稱為**函式庫**（library）的東西，拿去與他人共享。共享的函式庫生態，也有助於塑造網路的增長趨勢。
+在 JavaScript 誕生的 1996 年，它只有為當時由靜態文檔組成的網頁提供交互性和興奮感的作用。但之後網路漸漸從*拿來看*，變成*拿來用*了。JavaScript 慢慢地紅了起來，JavaScript 開發者也開始針對自己碰上的問題，寫出了能解決問題的工具、接著包成能複用的工具包。他們就能把這個被稱為**函式庫**（library）的東西，拿去與他人共享。共享的函式庫生態，也有助於塑造網路的增長趨勢。
```
Comment (irvin) [inline files/zh-tw/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.md:32]: 參考[原文](https://github.com/mdn/content/blob/6033ca2f22833bbfee06981bb5a82e271525664e/files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.md?plain=1#L38)整段一併調整

`When JavaScript debuted in 1996, it added occasional interactivity and excitement to a web that was, up until then, composed of static documents. The web became not just a place to _read things_, but to _do things_. JavaScript's popularity steadily increased. Developers who worked with JavaScript wrote tools to solve the problems they faced, and packaged them into reusable packages called **libraries**, so they could share their solutions with others. This shared ecosystem of libraries helped shape the growth of the web.`

```suggestion
在 JavaScript 誕生的 1996 年，它的作用僅是為主要由靜態文件組成的網頁，添加些許的互動和興奮感。但網路漸漸從_閱讀_的場所，轉變為_做事_的地方。JavaScript 慢慢地紅了起來，開發者也開始撰寫工具，以解決他們遭遇的問題，並打包成易於重複使用的**函式庫**（library），以便與他人分享。這個共享函式庫的生態，也同時推進了網路的增長。
```

**Review by irvin — CHANGES_REQUESTED** [review]
請參考

File: files/zh-tw/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.md (hunk 1 above)
Comment (DevRickLin) [reply files/zh-tw/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.md:32]: 我應該使用 --amend 覆蓋之前的 commit，還是在修改後提交新的 commit ?

Comment (yin1999) [reply files/zh-tw/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.md:32]: 提交新的 commit 即可

Comment (DevRickLin) [reply files/zh-tw/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.md]: 已經提交了該段落的翻譯，並進行了部分的潤色。

File: files/zh-tw/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ slug: Learn/Tools_and_testing/Client-side_JavaScript_frameworks/Introduction
 
 ## 一段簡短的歷史
 
-在 JavaScript 誕生的 1996，它的作用就只有針對網頁，提供些許的互動和興奮。但之後網路漸漸從*拿來看*，變成*拿來用*了。JavaScript 慢慢地紅了起來，JavaScript 開發者也開始針對自己碰上的問題，寫出了能解決問題的工具、接著包成能複用的工具包。他們就能把這個被稱為**函式庫**（library）的東西，拿去與他人共享。共享的函式庫生態，也有助於塑造網路的增長趨勢。
+在 JavaScript 誕生的 1996 年，它的作用僅僅是為當時由靜態文檔組成的網頁，提供些許的交互性和興奮感。而當網路漸漸從*閱讀＊的場所，成為*做事*的場所，JavaScript 也慢慢地紅了起來。JavaScript 開發者們撰寫了許多工具來解決自己遭遇的問題、並打包成能複用的工具包，稱為**函式庫**（library），以便和他人共享自己的解決方案。共享的函式庫生態也推進了網路的增長。
```
Comment (irvin) [inline files/zh-tw/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.md:32]: 繁體中文中的 interactive 翻譯為「互動」而非「交互性」。
「複用」之處也較「重複使用」更少見。此兩處我直接修改。
```suggestion
在 JavaScript 誕生的 1996 年，它的作用僅僅是為當時由靜態文檔組成的網頁，提供些許的互動和興奮感。而當網路漸漸從_閱讀_的媒介，成為_做事_的場所，JavaScript 也慢慢地紅了起來。JavaScript 開發者們撰寫了許多工具，來解決自己遭遇的問題，並打包成能重複使用的工具包，稱為**函式庫**（library），以便和他人共享自己的解決方案。函式庫共享的生態，也推進了網路的增長。
```

**Pull request merged**